src/project-manager.ts

```diff
@@ -892,9 +891,14 @@ export class ProjectConfiguration {
 			return;
 		}
 
+		// paths have been forward-slashified by TS (see toUnixPath below)
+		// TODO: consider moving expectedFilePaths out of LanguageServiceHost?
```

Comment:
Funny, I always thought TS required `expectedFilePath` to be implemented. Turns out it doesn't and it's not even used inside the class. We can totally move it to `ProjectConfiguration`.